README.md

```diff
@@ -37,6 +37,14 @@ By default Docker images built using clearwater-docker will pull the latest stab
 
 ![alt text](docs/images/clearwater-docker_in_scope.jpg "Clearwater-Docker dispayed in Scope")
 
+## Limitations
+
+Note that scaling of Docker deployments is a work in progress and there are currently a number of known issues...
+
+* Homestead-prov and Ellis don’t load balance across multiple Cassandra nodes.
+
+* In general deleting pods from storage clusters (Cassandra, Chronos or Astaire) is not supported.  Pods that are deleted will not get removed from the clusters and the clusters will end up broken.  The exception is when deployed under Kubernetes -- in this scenario Chronos and Astaire pods can be terminated, so long as this is done gracefully such that their prestop event hook is executed.  This means that Astaire and Chronos clusters (under Kubernetes) can be dynamcially scaled up and down.
```

Comment:
Worth mentioning that you shouldn't do both at the same time? or in quick succession?

Reply:
Add below.